documentation-site/pages/blog/getting-started-with-base-web/index.mdx

```diff
@@ -4,7 +4,7 @@ import metadata from './metadata.json'
 
 export default Layout;
 
-<BlogImage style={{marginTop: '20px'}} src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" />
+<BlogImage style={{marginTop: '20px'}} full src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" caption='Base UI by Uber Design'/>
```

Comment:
```suggestion
<BlogImage style={{marginTop: '20px'}} full src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" caption='Base Web by Uber Design'/>
```

Reply:
I wasn't sure about this one as the actual title on dribbble is Base UI. Base Web being the implementation of Base UI right?

Reply:
I think they made it just `Base` that has `BaseWeb`/`BaseMobile` @gergelyke is that correct?

Reply:
that's correct - everything we work on in this repository is Base Web

Reply:
The point I'm getting at is that the actual `dribble` shot title is `Base UI`. It's not a picture of `Base Web`. The suggested change is to update the caption.

Reply:
Base UI got rebranded to Base Web, so they are the same things, but moving forward, we'll use the Base Web name. Does it make sense?

Reply:
Yeah that makes sense. I wasn't aware of that 👍